dashboard/test/ui/features/artist.feature

```diff
@@ -20,6 +21,7 @@ Scenario: Winning the first level
   And I wait to see ".congrats"
   And element ".congrats" is visible
   And I press "continue-button"
+  Then I close the dialog
```

Comment:
The wait to find the desired element appears to do an initial check without waiting to see if the element is already present.

I say that because I recently added the code to wait for the element to appear, measured the before and after times, and found they were pretty much the same.  Prior to that change, the test would fail immediately if the pressed element was not already on the page, which contributed to flakiness.